README.md

```diff
@@ -1,2 +1,3 @@
 # fiware-health
 Program to check the functionality of each FIWARE Lab node.
```

Comment:
umm that is not my README. Mine is in fiware-region-sanity-tests/README.rst.
I have added the code in a subdir called 'fiware-region-sanity-tests'. Do you think better put it in the main dir?